src/VS.Web.CG.Mvc/View/ViewGeneratorModel.cs

```diff
@@ -15,6 +15,9 @@ public class ViewGeneratorModel : CommonCommandLineModel
         [Option(Name = "partialView", ShortName = "partial", Description = "Generate a partial view, other layout options (-l and -udl) are ignored if this is specified")]
         public bool PartialView { get; set; }
 
+        [Option(Name = "bootstrapVersion", ShortName = "b", Description = "Specify the bootstrap version. Valid values: '3', '4'. Default is 4.")]
```

Comment:
Looks like this description shows up in ~3 places, maybe it should be a resource?

Reply:
I thought about making it a resource, but decided to stick with the status quo of not making the command line arg descriptions be in resources.